tasmota/xdrv_23_zigbee_A_impl.ino

```diff
@@ -1344,7 +1374,7 @@ void ZigbeeShow(bool json)
     if (zigbee_num > 255) { zigbee_num = 255; }
 
     // Calculate fixed column width for best visual result (Theos opinion)
-    const uint8_t px_batt = 30;                         // Battery icon is 20px, add 10px as separator
+    const uint8_t px_batt = 50;                         // Battery icon is 20px, add 30px for separator + buffer
```

Comment:
Why do you need to increase the size?

Reply:
I increased this ever so slightly to "fix" the table increasing/decreasing in size, which is related to your comment about seconds resolution. The timestamps would cause the table width to increase / decrease. This was a small attempt to remove that.